www/client/containers/DataVisualization/index.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchComponentData } from './actions';
+import ComponentPage from './ComponentPage';
+
+class ComponentContainer extends React.Component {
```

Comment:
Same comment here as above since this is the DataVisContainer

Reply:
Just refer to the Components structs.